auth/services/selfsigned/web/spec.yaml

```diff
@@ -0,0 +1,125 @@
+openapi: "3.0.0"
```

Comment:
Because it is an internal service and should not be documented. Just like the IRMA endpoints are not documented. We don't want people to build clients for it since its only http pages.